project.md

```diff
@@ -4,30 +4,29 @@ Edit this file to include your project idea pitch for the HighTechU Academy.
 
 ## Project Title
 
-[DELETE ME - ADD YOUR PROJECT TITLE HERE]
+Naloxone Tracker App
 
 Example - Colour Picker
 
 ## Project Mini Pitch
 
-[DELETE ME - ADD YOUR PROJECT MINI PITCH HERE]
+This app could potentially save lives by connecting those in crisis with those that have the tools to help.
 
-Example - This colour picker will revolutionaize the way colours are picked for the web.
+Example - This colour picker will revolutionize the way colours are picked for the web.
 
 ## Project Description
 
-[DELETE ME - ADD YOUR PROJECT DESCRIPTION HERE]
-
+The app would keep track of who in your area has a Naloxone kit on their person and sends out a notification to those members when someone around them broadcasts a request. This would prevent not being able to take action in those crucial moments between noticing someones overdosing and first responders getting there, lots of teens carry kits in their bags at all times and have had the proper training but the problem occurs when you don't know who around your location might have one on them. BC has been having an opioid crisis and more people are getting sent to the hospital for overdosing than ever, though this is not specifically a teen issue this would give teenagers the opportunity to help people at parties, on the street, in their homes or hypothetically anywhere that people are using phones.
 Example - The app will be web based and allow users to pick a colour with the help of a colour wheel.
 
 ## Project Audience
 
-[DELETE ME - ADD YOUR PROJECT AUDIENCE HERE]
+The audience would be anyone in the vancouver island area especially those with Naloxone kit training, people who frequently go out, those with high risk family and friends who don't have their own kit, people in areas that first responders take longer to reach, people who are often in areas like downtown where you can sometimes see people in crisis and those who don't know anyone in their social circle or area with a kit. This app would reach a fairly broad audience in the Vancouver island community because you never know when it could be useful to know where the closest person with a Naloxone kit is.
 
 Example - The audience would be frontend developers and designers looking to pick colours for their work.
 
 ## Project Inspiration
 
-[DELETE ME - ADD YOUR PROJECT INSPIRATION HERE]
+Inspiration - Find My Friends but for medical notifications 
 
 Example - This project idea was inspired by the following websites: list of website.
```

Comment:
Hey Anjali!

This is a really solid app! Thank you for sharing your app idea with the Cohort.